### DISCUSSION TITLE
Copy on select

### DISCUSSION THREAD

**Discussion by dotancohen** [plateaukao/einkbro#287] [Ideas]
I must copy text very often from einkbro to other applications. It would be nice if einkbro were to copy text to the clipboard immediately, instead of having to select Copy.

This would be akin to the X Window System's primary keyboard, in which one can simply select text to copy it.

**Comment by gety9** [comment]
@dotancohen, hi. I copy text all the time as well and sort of agree that instant copy will be great, but then how will other on select operations can be performed? (translate, search, highlight, external app, etc)

**Reply by wasu-code** [reply]
I guess Copy is executed the moment text is selected but the popup with other c
actions still appears (just as it does now).

Similar feature is available in MoonReader and I can assure you it is very convenient.  
In MoonReader it doesn't copy but send intents to application you have choosen in settings (eg. Translator, dictionary etc) what makes translating very fast and easy.  

It would be great to allow configuring what action should be performed immediately (whether it's copying, transalting or any other)
